test/test_compiler.py

```diff
@@ -24,17 +26,18 @@
     PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
 )
 from .test_declare_v2 import assert_declare_v2_accepted, load_cairo1_contract
-from .util import (
-    DevnetBackgroundProc,
-    devnet_in_background,
-    read_stream,
-    terminate_and_wait,
-)
+from .util import DevnetBackgroundProc, read_stream, terminate_and_wait
 
-SPECIFIED_MANIFEST = os.getenv("CAIRO_1_COMPILER_MANIFEST")
-if not SPECIFIED_MANIFEST:
+CAIRO_1_COMPILER_MANIFEST = os.getenv("CAIRO_1_COMPILER_MANIFEST")
+if not CAIRO_1_COMPILER_MANIFEST:
     raise KeyError("CAIRO_1_COMPILER_MANIFEST env var not set")
 
+# kind of a hack that relies on the old configuration of working with the manifest
+CAIRO_1_COMPILER_REPO = os.path.dirname(CAIRO_1_COMPILER_MANIFEST)
```

Comment:
Can this be explained more deeply? Why `CAIRO_1_COMPILER_REPO = os.path.dirname(CAIRO_1_COMPILER_MANIFEST)` ? It will be always like this? Is it the assumption that might change in the future?

I would also not use the word "hack". Just explained it more deeply.

Reply:
Addressed here: https://github.com/0xSpaceShard/starknet-devnet/pull/471/commits/13accea151cc3076302b16b11ffe319c8767c9fd